scrapper.py

```diff
@@ -44,18 +75,50 @@ def get_search_urls(self):
         pass
 
 
+class ArticleParser:
+    def __init__(self, full_url, identification):
+        self.article_url = full_url
+        self.article_id = identification
+        self.article = ''
```

Comment:
it should be an instance of Article class

Reply:
done